models/attachment.go

```diff
@@ -20,15 +20,15 @@ import (
 
 // Attachment represent a attachment of issue/comment/release.
 type Attachment struct {
-	ID        int64  `xorm:"pk autoincr"`
-	UUID      string `xorm:"uuid UNIQUE"`
-	IssueID   int64  `xorm:"INDEX"`
-	CommentID int64
-	ReleaseID int64 `xorm:"INDEX"`
-	Name      string
-
-	Created     time.Time `xorm:"-"`
-	CreatedUnix int64
+	ID            int64  `xorm:"pk autoincr"`
+	UUID          string `xorm:"uuid UNIQUE"`
+	IssueID       int64  `xorm:"INDEX"`
+	CommentID     int64
+	ReleaseID     int64 `xorm:"INDEX"`
+	Name          string
+	DownloadCount int64
```

Comment:
It's better to do that. I think we can begin to ask all schema changes should use https://github.com/go-xorm/xorm/tree/master/migrate